src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,21 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodowithUser } from '../types';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
please, prefer the interface